spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/reload/ConfigReloadProperties.java

```diff
@@ -52,7 +54,7 @@
 	/**
 	 * Sets the polling period in milliseconds to use when the detection mode is POLLING.
 	 */
-	private Long period = 15000L;
+	private Duration period = Duration.ofMillis(15000L);
```

Comment:
Need to change the javadoc, since it's not millis anymore

Reply:
Done